tests/playground.html

```diff
@@ -465,6 +470,21 @@
     }
   };
   menuOptions.push(screenshotOption);
+
+  // Adds a default-sized workspace comment to the workspace.
+  var workspaceCommentOption = {
```

Comment:
Instead of creating a completely new option for the playground, can we make use of: 
```
Blockly.ContextMenu.workspaceCommentOption
```

Reply:
Totally didn't know this existed! Thanks for the tip :D